README.md

```diff
@@ -1,14 +1,14 @@
-# Verimatrix App Shield
+# Verimatrix XTD and Counterspy
 
-This action integrates Verimatrix App Shield into your GitHub build workflow. It automates
+This action integrates Verimatrix Extended Threat Defense and Counterspy into your GitHub build workflow. It automates
 the protection of your Android and iOS apps so you can run it whenever a new version 
 of your application is built. 
 
-Verimatrix App Shield is a zero-code in-app protection service. It protects your apps from 
+[Verimatrix XTD and Counterspy](https://portal.platform.verimatrixcloud.net/) is a zero-code in-app protection service. It protects your apps from 
 reverse engineering and attack through a layered security approach. Protection layers include
 obfuscation, environmental checks and binary integrity checks. 
 
-A [Verimatrix App Shield](https://appshield.verimatrixcloud.net/) Pro account is required to use this Action. 
+All **XTD** accounts support this action, while for **Counterspy** you need a standard account. 
```

Comment:
"subscription tiers" instead of accounts, and "Standard subscription"(capital S)  instead of "standard account".